api/swagger.yaml

```diff
@@ -794,6 +844,159 @@ paths:
         "200":
           description: "ok"
       x-swagger-router-controller: "Notifications"
+  /classrooms:
+    get:
+      tags:
+      - "classroom"
+      summary: "Get all classrooms."
+      description: ""
+      operationId: "getAllClassrooms"
+      produces:
+      - "application/json"
+      parameters: []
+      responses:
+        "200":
+          description: "ok"
+          schema:
+            type: "array"
+            items:
+              $ref: "#/definitions/Classroom"
+      x-swagger-router-controller: "Classroom"
+  /classroom:
+    post:
+      tags:
+      - "classroom"
+      summary: "Create classroom"
+      operationId: "classroomPOST"
+      produces:
+      - "applciation/json"
```

Comment:
Should be `application/json`.

Reply:
Also quite a few of the same typo from here on, just find and replace `applciation` with `application`.

Reply:
@william-gooch  typo is fixed.